enclave/sgx/collateralinfo.c

```diff
@@ -169,6 +208,133 @@ oe_result_t oe_get_sgx_quote_verification_collateral(
     return result;
 }
 
+void oe_prealloc_quote_verification_collateral_args(
+oe_get_sgx_quote_verification_collateral_args_t* buf, oe_get_sgx_quote_verification_collateral_args_t* default_sizes)
```

Comment:
nit: This doesn't seem to follow the style guide. Running scripts/format-code on each of these files should fix any issues.